src/WorkItemMigrator/JiraExport/JiraItem.cs

```diff
@@ -182,18 +187,22 @@ private static JiraRevision BuildCommentRevision(Comment c, JToken rc, JiraItem
             };
         }
 
-        private static void UndoAttachmentChange(RevisionAction<JiraAttachment> attachmentChange, List<JiraAttachment> attachments)
+        private static bool UndoAttachmentChange(RevisionAction<JiraAttachment> attachmentChange, List<JiraAttachment> attachments)
         {
+            bool result = false;
```

Comment:
Same here, I would split out to specific functions instead. sure, the first if can stay but i would return false right away.
In the else i would refactor out to a method RemoveAttachment that would handle remove part and would return boolean, eg: 

if(RemovedType)
reutrn false;
return RemoveAttachmentChange(param1, param2);

Reply:
Resolved